docs/plugins/repository-azure.asciidoc

```diff
@@ -154,7 +154,7 @@ PUT _snapshot/my_backup2
     "settings": {
         "container": "backup-container",
         "base_path": "backups",
-        "chunk_size": "32m",
+        "chunk_size": "32MB",
         "compress": true
```

Comment:
Curious if both notations work...but being consistent with the description makes sense.

Reply:
I confirmed with @jen-huang that the notation needs unit and number - https://github.com/elastic/elasticsearch/issues/41591

Reply:
FWIW, it does need both unit and number, but it appears that either `m` or `mb` as the unit works. Similarly, `g` or `gb` works as well. @elastic/es-distributed team, do we have a preference regarding this?

Reply:
Per Armin:
> I don't think there's a difference there, looking at the code both versions are treated the same in `ByteSizeValue` and there's no deprecation or anything

As far as docs are concerned, I think we can keep this change that follows the doc style guide (i.e. `MB`). Thanks @wchrisdean!